src/modules/update_project/app/update_project_usecase.py

```diff
@@ -0,0 +1,50 @@
+from typing import List
+from src.shared.domain.entities.member import Member
+from src.shared.domain.entities.project import Project
+from src.shared.domain.enums.role_enum import ROLE
+from src.shared.domain.repositories.action_repository_interface import IActionRepository
+from src.shared.helpers.errors.domain_errors import EntityError
+from src.shared.helpers.errors.usecase_errors import NoItemsFound, ForbiddenAction
+
+
+class UpdateProjectUsecase:
+    def __init__(self, repo: IActionRepository):
+        self.repo = repo
+
+    def __call__(self, code, new_name: str, new_description: str, new_po_RA: str, new_scrum_RA: str, new_photos: List[str]) -> Project:
```

Comment:
é um hard update ne, não gosto mto disso mas ta suave, futuro a gente arruma se for o caso

(só comentario sem nada pra reclamar aqui)

Reply:
desci mais um pouco, não é um hardupdate porque no repo vc só atualzia o que mudou...

controller não devia levantar erro se for none

Reply:
Controller não levanta mais erro